tests/test_pca.py

```diff
@@ -110,7 +111,8 @@ def array_type(request: pytest.FixtureRequest) -> ArrayType:
     return request.param
 
 
-SVDSolver = Literal["auto", "full", "arpack", "randomized", "tsqr", "lobpcg"]
```

Comment:
What happened to `tsqr` in the actual implementation?

Reply:
Never there?

Reply:
Doesn’t it work like advertised?

It’s in `SvdSolvPCADaskML` so if you have a dense dask array, you can use it

Reply:
Apologies, was looking locally with a filter on